job-tech-workshop (#341)
CodeYourFuture/recap-workshops: jobs-in-tech/README.md

### Let's do some research 
1.  Make pairs or small groups of 3-4- and research waht are the jobs availabe in tech indsutry. You can also use these [CompTIA Career Pathways]{https://www.comptia.org/en/explore-careers/job-roles/?page=1&category=all) or 
[TechCareerExplorer](https://www.careerexplorer.com/careers/?page=23&industry=42) as reference. Each group should chose one of these groups to research: 
- Engineering roles 
- Low code roles 
- Tech jobs without coding
- IT jobs 
Write 1 role per post it. (10 min)

2. Choose a wall (or, if online, use a Miro board or similar), and walk through each post it. See if you can explain the role - and if you can't, ask if anyone in the group can. (15 min)

3. Pick one role from each group and let's identify more about it: (15 min)
   🎯 Job Title: 
   🧠 What do they actually do all day?
   💬 Tools or software they use (Google it!)
   📈 How to get this job: Degree? Certifications? Courses?
   📋 Skills needed (soft and hard):
   🔍 Find a real job post and copy 1 key requirement:

4. Present the role back to the group.

<details>

<summary>Hint for volunteers</summary>

Make sure we've covered:

> * Engineering roles: Software developer, Quality Engineer, System Reliability Engineer 

Reviewer: ```suggestion
* Engineering roles: Software developer, Quality Engineer, Site Reliability Engineer 
```

* Low code roles: Low-code Dev and No-code Dev, Wordpress Developer, Database administrator
* Jobs in tech without coding: Scrum Master, Manual tester, Business Analyst, 
* IT Jobs: IT Analyst, IT Support

</details>

## Which one is for me?

As we said in the start of this workshop, software developer is one of the many paths in the tech. And CodeYourFuture wants to support you in your chosen career.

<details>

<summary>Hint for volunteers</summary>

> Focus on transferrable skills and new skills, as well as what is their reality and availablity to get ready for it

Reviewer: ```suggestion
Focus on transferable skills and new skills, as well as what is their reality and availability to get ready for it
```

</details>

Get together in pairs and talk about your own career path considering what you learned today. Use the below scripts to reflect on (5 min)
- What are your good professional skills, the ones everyone always praises you about?
- From the roles we discussed, which one are you most interested and why?
- What will be your next steps to focus on getting this role?
- What will be the blockers I will have?

Write down your career action plan, which you can discuss further with your Career mentor:

> 💼 Job I want to explore more:
> 🎓 Skills I need to learn:
> 📚 Free resources I’ll start with:
> 🧑‍🤝‍🧑 A person/community I’ll reach out to:
> 📆 3 Things I’ll do this week:

Reviewer: ```suggestion
- 💼 Job I want to explore more:
- 🎓 Skills I need to learn:
- 📚 Free resources I’ll start with:
- 🧑‍🤝‍🧑 A person/community I’ll reach out to:
- 📆 3 Things I’ll do this week:
```
